src/GitVersionCore/Model/GitVersionOptions.cs

```diff
@@ -9,23 +9,23 @@ namespace GitVersion
     public class GitVersionOptions
     {
         private Lazy<string> dotGitDirectory;
-        private Lazy<string> projectRootDirectory;
-        private Lazy<string> dynamicGitRepositoryPath;
+        private Lazy<string> gitRepositoryWorkingDirectory;
 
         public GitVersionOptions()
         {
+            WorkingDirectory = System.Environment.CurrentDirectory;
             dotGitDirectory = new Lazy<string>(this.GetDotGitDirectory);
-            projectRootDirectory = new Lazy<string>(this.GetProjectRootDirectory);
-            dynamicGitRepositoryPath = new Lazy<string>(this.GetDynamicGitRepositoryPath);
+            gitRepositoryWorkingDirectory = new Lazy<string>(this.GetRepositoryWorkingDirectory);
         }
 
+        public string[] Args { get; set; }
         public string WorkingDirectory { get; set; }
-
         public string DotGitDirectory => dotGitDirectory.Value;
-        public string ProjectRootDirectory => projectRootDirectory.Value;
```

Comment:
ProjectRootDirectory --> renamed to GitRepositoryWorkingDirectory for selfish reasons.